app/models/concerns/searchable.rb

```diff
@@ -20,8 +20,35 @@ def search(name_or_phone_str, lines = LINES)
       sort_and_limit_patient_matches(all_matching_names, all_matching_phones, all_matching_identifiers)
     end
 
+    def search_users(name_str)
```

Comment:
I'd love to be able to search on email too, if that's something that can be in the cards! But I'm okay with rolling that out separate since there's already a lot swirling in this pr.